packages/components/format-date/src/format-date.stories.ts

```diff
@@ -0,0 +1,160 @@
+import { type Meta, type StoryObj } from '@storybook/web-components';
+import { type TemplateResult, html, nothing } from 'lit';
+import '../register.js';
+import { type FormatDate } from './format-date.js';
+import { formatDateTime } from './format-date-time';
+
+interface Props extends FormatDate {
```

Comment:
`type Props = Pick<FormatDate, 'date' | 'locale' | etc> & { fallback: TemplateResult };`

At the moment, `Props` has *all* properties of LitElement etc.